pgsql/CMakeLists.txt

```diff
@@ -16,11 +20,16 @@ set ( PC_INSTALL_EXENSIONS
   "${PROJECT_BINARY_DIR}/pgsql/pointcloud.control"
   )
 
-configure_file(
-    pointcloud.sql.in
-    "${PROJECT_BINARY_DIR}/pgsql/pointcloud--${POINTCLOUD_VERSION}.sql"
+configure_file(pointcloud.sql.in pointcloud.sql)
+configure_file(li3ds.sql.in li3ds.sql)
```

Comment:
Isn't this going to create untracked files in the repo?

Also, I would like to find a better way, compatible with upstream.